templates.dist/estate/parkingLot/ParkingLot.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
If you add the parking lot field to an estate list, the CSS will be for the detail view.

![GUI](https://user-images.githubusercontent.com/90763364/161773446-0efd66cd-be35-4672-a4b0-d0e8e8e672b7.png)

Therefore please remove this file and inline its code into the templates where it is used. Also make sure that the classes are the same as the rest of the fields in that template.